commit 6025237206f2cfbe540edaa29982044dc5136952 (refs/changes/20/820/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-31T12:05:30-07:00 (2 years, 11 months ago)
    
    TOOLS-1595 separate imgapi's pkgsrc packages: the stock set + imgapi-specific (no functional change)

diff --git a/targets.json.in b/targets.json.in
index 40e9c35..7fcdc90 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -588,16 +588,16 @@ cat <<EOF
     "pkgsrc": [
         "coreutils-8.23nb2",
         "curl-7.47.1",
-        "dateutils-0.3.1nb1",
-        "grep-2.22",
         "gsed-4.2.2nb4",
         "patch-2.7.5",
-        "xz-5.2.2",
+        "sudo-1.8.15",
+
+        "dateutils-0.3.1nb1",
         "haproxy-1.6.2",
         "openssl-1.0.2j",
         "stud-0.3p53nb5",
-        "sudo-1.8.15",
-        "the_silver_searcher-0.31.0"
+        "the_silver_searcher-0.31.0",
+        "xz-5.2.2"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-imgapi.git"}
